rpcs3/Emu/Cell/PPUModule.cpp

```diff
@@ -87,7 +88,7 @@ ppu_static_function& ppu_module_manager::access_static_function(const char* _mod
 
 ppu_static_variable& ppu_module_manager::access_static_variable(const char* _module, u32 vnid)
 {
-	auto& res = access().at(_module)->variables[vnid];
+	auto& res = ppu_module_manager::staticModuleMap.at(_module)->variables[vnid];
```

Comment:
```suggestion
	auto& res = staticModuleMap.at(_module)->variables[vnid];
```

Reply:
You don't feel that leaving in the full scoping makes it 'more' apparent that it's a static member variable and not just 'any old' member variable?

I'll fix up the others, and prepare to push again (and will just wait for your feedback on this one)